### PR TITLE
MINIFI-CPP-1203 - Add linter recommended missing whitespaces

### PR DESCRIPTION
**Pull request by hunyadi-dev**
List of errors corrected:
```
    123 Missing space before ( in if(  [whitespace/parens] [5]
     90 Missing space after ,  [whitespace/comma] [3]
     71 Missing space before {  [whitespace/braces] [5]
     24 Missing space before ( in for(  [whitespace/parens] [5]
     10 Missing spaces around =  [whitespace/operators] [4]
      9 Missing space around colon in range-based for loop  [whitespace/forcolon] [2]
      3 Missing space before ( in while(  [whitespace/parens] [5]
      2 Missing spaces around <  [whitespace/operators] [3]
      1 Missing spaces around <<  [whitespace/operators] [3]
      1 Missing space before ( in switch(  [whitespace/parens] [5]
      1 Missing space before else  [whitespace/braces] [5]
```

Turn off ignore-whitespace to review the PR.

### CODE REVIEW

**Comment by szaszm** [comment]
I'm not a fan of space before the colon in range-based for loops.
Why does the linter check pass without these changes? Are these rules disabled?

File: extensions/sftp/tests/PutSFTPTests.cpp [hunk 1]
```diff
@@ -809,7 +809,7 @@ TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP batching does not fail even if on
 }
 
 TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP put large file", "[PutSFTP]") {
-  std::mt19937 rng(std::random_device{}());
+  std::mt19937 rng(std::random_device {}());
```
[the remark this answers is not in the document]
Comment (hunyadi-dev) [reply extensions/sftp/tests/PutSFTPTests.cpp:812]: I replaced it with `// NOLINT` but I am not sure it helps readibility.

File: extensions/windows-event-log/wel/WindowsEventLog.h [hunk 2]
```diff
@@ -87,36 +87,36 @@ class WindowsEventLogMetadata {
...
+        {EVENT_TYPE, "EVENT_TYPE"}, 
         {TASK_CATEGORY, "TASK_CATEGORY"},
-        {LEVEL,"LEVEL"},
-        {KEYWORDS,"KEYWORDS"},
-        {USER,"USER"},
-        {COMPUTER,"COMPUTER"}
+        {LEVEL, "LEVEL"}, 
+        {KEYWORDS, "KEYWORDS"}, 
+        {USER, "USER"}, 
+        {COMPUTER, "COMPUTER"}
```
[the remark this answers is not in the document]
Comment (hunyadi-dev) [reply extensions/windows-event-log/wel/WindowsEventLog.h:100]: Good catch, It was an issue with my replacement script, I figured that if there was no space after a comma in any reported line, there should be, but this was a wrong assumption.

File: extensions/windows-event-log/wel/WindowsEventLog.h [hunk 3]
```diff
@@ -87,36 +87,36 @@ class WindowsEventLogMetadata {
...
-        {"EVENT_TYPE",EVENT_TYPE},
-        {"LEVEL",LEVEL},
-        {"KEYWORDS",KEYWORDS},
-        {"USER",USER},
-        {"COMPUTER",COMPUTER}
+        {"EVENT_TYPE", EVENT_TYPE}, 
+        {"LEVEL", LEVEL}, 
+        {"KEYWORDS", KEYWORDS}, 
+        {"USER", USER}, 
+        {"COMPUTER", COMPUTER}
```
[the remark this answers is not in the document]
Comment (hunyadi-dev) [reply extensions/windows-event-log/wel/WindowsEventLog.h:119]: same as above.

**Comment by hunyadi-dev** [comment]
> I'm not a fan of space before the colon in range-based for loops.
> Why does the linter check pass without these changes? Are these rules disabled?

1. I am not a fan either, but this is a listed rule in the [google style guide](https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace).
2. Yes, these files are not yet checked.

**Review by martinzink — APPROVED** [review]
LGTM

**Review by adamdebreceni — APPROVED** [review]

**Pull request closed without merging**